pkgs/applications/misc/girara/default.nix

```
@@ -19,19 +19,16 @@
 
 stdenv.mkDerivation rec {
   pname = "girara";
-  version = "0.4.3";
+  version = "0.4.5";
 
   outputs = [
     "out"
     "dev"
   ];
 
-  src = fetchFromGitLab {
-    domain = "git.pwmt.org";
-    owner = "pwmt";
-    repo = "girara";
-    rev = version;
-    hash = "sha256-/bJXdLXksTxUFC3w7zuBZY6Zh7tJxUJVbS87ENDQbDE=";
+  src = fetchzip {
```

If we fetch a tarball then please
```suggestion
  src = fetchurl {
```